fabric-biome-api-v1/src/main/java/net/fabricmc/fabric/api/biome/v1/BiomeSelectors.java

```diff
@@ -104,6 +105,15 @@ private BiomeSelectors() {
 		return context -> context.getBiome().getCategory() == Biome.Category.THEEND;
 	}
 
+	/**
+	 * Returns a biome selector that will match all biomes in the given tag.
+	 *
+	 * @see net.fabricmc.fabric.api.tag.TagFactory#BIOME
+	 */
+	public static Predicate<BiomeSelectionContext> tag(Tag<Biome> tag) {
```

Comment:
should we enforce tag.identified to prevent accidentally passing tags that expire on reload?

Reply:
I don't think that it's matter since biome modifications only called once on server setup.

Reply:
do modification apply to biomes after each reload? if yes, on setup, a persistent (identified) tag should be passed to address biome tag changes in reloads, as you don't have a chance to update the tag reference manually later. another supplier of tag may work too.

Reply:
An instance: say some the biome tag selector test accidentally just acquired a tag from the tag group for the biome tags; the tag definitely don't update across resource reloads. if a data pack that adds swamp to the tag is enabled, the predicate still will use the outdated tag, and will return `false` for a swamp biome context.

Using `Tag.Identified<Biome>` is just an extra level of safeguard for good practices, which I think its benefits outweigh its restrictions.

Reply:
All modifications will only be applied once, right after the dynamic registry is loaded (server setup.) The function won't even get called on `/reload`. Even if it is somehow called multiple times, the individual biome can only be modified once anyway.

Reply:
Hmm, I wonder if this weird fact should be documented or not